spec/features/admin/dashboard/index_spec.rb

```diff
@@ -192,4 +192,10 @@
       end
     end
   end
+
+  describe 'presence of github information' do
+    scenario 'visitor sees repo name, contributors, commits, and total pull requests on page' do
+      allow
```

Comment:
I think stubbing our api call tests would be awesome so we could actually run them every time